newIDE/app/public/external/yarn/yarn-main.js

```diff
@@ -59,23 +69,22 @@ ipcRenderer.on('yarn-open', (event, receivedData) => {
   yarn.document.getElementsByClassName('app-search')[0].style = 'right: 45px';
   saveToGdButton.style = 'padding-left: 30px;';
 
-  // Process the json file to open, if any.
-  if (fileExists(receivedData.resourcePath)) {
-    receivedData.externalEditorData = fs
-      .readFileSync(receivedData.resourcePath, 'utf8')
-      .toString();
-
-    yarn.data.loadData(receivedData.externalEditorData, 'json', true);
-    electronWindow.setTitle(
-      'GDevelop Dialogue Tree Editor (Yarn) - ' + receivedData.resourcePath
-    );
+  yarn.data.editingPath('');
```

Comment:
why is this empty?

Reply:
There is no notion of 'path' for Yarn, because we're editing files that may not have a path. I'm not sure why Yarn would use it, I kept it to be safe.

Reply:
This seems harmless: https://github.com/blurymind/YarnClassic/search?q=editingPath